test/datetime/create.test.js

```diff
@@ -571,7 +571,7 @@ test("DateTime.fromObject accepts a locale string with weird junk in it", () =>
     });
 
     expect(res.locale).toBe("be");
-    expect(res.outputCalendar).toBe("gregory");
+    expect(res.outputCalendar).toBe("coptic");
```

Comment:
On my system (macOS 10.14.3) the test fails with `outputCalendar` expecting **coptic** instead of **gregory**. Had no choice but to fix the test before committing.

More info:

```bash
 ~/@/l/luxon $ node -v
v11.13.0
```
Node compiled and installed with-full-icu support.